ext/storage_sources/azure_store/azure_connection.h

```diff
@@ -0,0 +1,25 @@
+#ifndef AZURECONNECTION
+#define AZURECONNECTION
+
+// Includes necessary for connection
+#include <azure/core.hpp>
+#include <azure/storage/blobs.hpp>
+
+#include <string>
+
+class AzureConnection {
+    public:
+    AzureConnection(const std::string &bucketName, const std::string &objPrefix = "");
+    int ListObjects(const std::string &prefix, std::vector<std::string> &objects,
```

Comment:
we do not need to worry about the prefix, batchsize, and list single yet. Let's remove them.